cmd/solvercli/cmd/settlements.go

```diff
@@ -0,0 +1,94 @@
+package cmd
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/skip-mev/go-fast-solver/ordersettler"
+	"github.com/skip-mev/go-fast-solver/shared/clientmanager"
+	"github.com/skip-mev/go-fast-solver/shared/keys"
+	"github.com/skip-mev/go-fast-solver/shared/lmt"
+	"github.com/skip-mev/go-fast-solver/shared/txexecutor/cosmos"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+)
+
+type chainPair struct {
+	source      string
+	destination string
+}
+
+type groupedSettlement struct {
+	amount *big.Int
+	count  int
+}
+
+var settlementsCmd = &cobra.Command{
+	Use:     "settlements",
+	Short:   "Show pending settlement amounts across chains",
+	Long:    "Show pending settlement amounts across chains",
+	Example: "solver settlements",
+	Run: func(cmd *cobra.Command, args []string) {
+		ctx := setupContext(cmd)
+		keysPath, err := cmd.Flags().GetString("keys")
+		if err != nil {
+			lmt.Logger(ctx).Fatal("Error reading keys path", zap.Error(err))
+		}
+
+		keyStoreType, err := cmd.Flags().GetString("key-store-type")
+		if err != nil {
+			lmt.Logger(ctx).Fatal("Error reading key-store-type", zap.Error(err))
+		}
+
+		keyStore, err := keys.GetKeyStore(keyStoreType, keys.GetKeyStoreOpts{KeyFilePath: keysPath})
+		if err != nil {
+			lmt.Logger(ctx).Fatal("Unable to load keystore", zap.Error(err))
+		}
+
+		cosmosTxExecutor := cosmos.DefaultSerializedCosmosTxExecutor()
+		cctpClientManager := clientmanager.NewClientManager(keyStore, cosmosTxExecutor)
```

Comment:
seems like we can just pass a nil value for the key store if it's irrelevant to the function of this command, or does that create an error?